extensions/amp-next-page/0.1/next-page-service.js

```diff
@@ -497,19 +516,57 @@ export class NextPageService {
 
   /**
    * Sets the specified document as active, updating the document title and URL.
-   * @param {!DocumentRef} documentRef Reference to the document to set as
-   *     active.
+   *
+   * @param {DocumentRef} ref Reference to the document to be activated
```

Comment:
since the call to this function is guarded by `if (ref && ref.amp)`, shouldn't it be non-nullable at this point?

Reply:
Correct, thanks for catching that!